tests/optimizer/test-function-eliminator-replace-array-value-output.js

```diff
@@ -0,0 +1,16 @@
+var asm = (function(global, env, buffer) {
+ "use asm";
+ function a() {
+  return 0;
+ }
+ function c() {
+  a();
+  return;
+ }
+ function d() {
+  a();
+  var f = [ a ];
```

Comment:
I assume this is checking for function tables? the array should be outside of a function scope, then, to make this valid asm.js.